scripts/check-pr-guardian-contract.sh

```
@@ -62,16 +79,17 @@ for pattern in \
   '(.createdAt | type == "string")' \
   '(.outdated | type == "boolean")' \
   '(.comments.pageInfo.hasNextPage | type == "boolean")' \
+  '(.comments.pageInfo.hasNextPage == false)' \
   '($comments.nodes | type == "array")' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)